pkg/skaffold/docker/parse_test.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestUnquote(t *testing.T) {
+	testutil.CheckDeepEqual(t, `scratch`, unquote(`scratch`))
+	testutil.CheckDeepEqual(t, `scratch`, unquote(`"scratch"`))
+	testutil.CheckDeepEqual(t, `scratch`, unquote(`""scratch""`))
+	testutil.CheckDeepEqual(t, `scratch`, unquote(`'scratch'`))
+	testutil.CheckDeepEqual(t, `scratch`, unquote(`''scratch''`))
+	testutil.CheckDeepEqual(t, `'scratch'`, unquote(`"'scratch'"`))
```

Comment:
this seems a little inconsistent, can this strip the single quotes too?

Reply:
It's not what docker does. I know... I tried to support every case that docker supports, no more

Reply:
ugh, ok. not great but that's the right thing to do